themes/gokarna/static/css/main.css

```diff
@@ -151,7 +151,7 @@ a:hover .feather-sun {
 /* FOOTER */
 .footer {
   bottom: 0;
-  border-top: 1px solid var(--light-header-bg-color);
+  border-top: 0.5px solid var(--light-header-divider-color);
```

Comment:
Honestly, it looks good without any border, much cleaner

Reply:
- If the page has no scroll then it looks better without any border as header does not have any shadow as well
- But if the page has a scroll, then it looks weird without any border on mobile.

Reply:
I don't care